Resources/public/js/views/services/plugins/ez-objectrelationsloadplugin.js

```diff
@@ -81,7 +200,17 @@ YUI.add('ez-objectrelationsloadplugin', function (Y) {
              */
             contentModelConstructor: {
                 value: Y.eZ.Content
-            }
+            },
+
+            /**
+             * Location constructor
+             *
+             * @attribute locationModelConstructor
+             * @type Y.eZ.Location
```

Comment:
I would also add hhe `@since` tag here (I hope yuidoc handle it correctly in such context)